resources/s3-buckets.go

```diff
@@ -140,7 +140,8 @@ func (e *S3Bucket) RemoveAllObjects() error {
 
 func (e *S3Bucket) Properties() types.Properties {
 	properties := types.NewProperties()
-	properties.Set("Name", e.name)
+	properties.Set("Name", e.name).
+	properties.Set("CreationDate", e.CreationDate())
```

Comment:
I'm fairly new to Go so was not sure if it should be `e.CreationDate()` or `e.CreationDate`. I would be grateful of a poke in the right direction 😄

Reply:
Same as [above](https://github.com/rebuy-de/aws-nuke/pull/772#discussion_r845460140)